catalog-info.yaml

```
@@ -976,3 +976,42 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: beats-xpack-agentbeat-package
+  description: Buildkite pipeline for packaging and publishing agentbeat
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-xpack-agentbeat-package
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: beats-xpack-agentbeat-package
+      description: Buildkite pipeline for packaging and publishing agentbeat
+    spec:
+      repository: elastic/beats
+      pipeline_file: ".buildkite/x-pack/agentbeat/pipeline.xpack.agentbeat.package.yml"
+      # todo release branched must be 8.14+
+      branch_configuration: "main 8.14 v8.14"
```

```suggestion
      branch_configuration: "main 8.14"
```

What about `8.15` and others?

maybe something like, I have not tested it out

`"main 8.1[4-9] 8.[2-9][1-9]"

---

That's a good point for future branches. You are right; we don't do anything with the tags. I guess it's mainly for consistency with the other pipelines. 

We need to revisit those `branch_configurations` and pipeline settings overall.

---

Thanks for the suggestion. Let's test this pattern then the publishing works